src/rules/unused_callbacks.erl

```diff
@@ -0,0 +1,15 @@
+%% @doc A rule to detect unused callbacks.
+%%      <p>The rule emits a warning if it can't find the callback's atom name
+%%      being used within a module. It will NOT emit a warning if an atom named
+%%      as a callback is being used, no matter for what that atom is used.</p>
```

Comment:
Since the docs are used as informative descriptions of each rule, I think we should clearly state the goal of the rule before diving into the implementation details and the caveats of _how_ it works.
In other words, I would add something along the lines of…

> This rule will check all callbacks defined in a module and find those that are not used anywhere in the module itself. The assumption is that if you define a callback for a behavior, your _generic module_ (where the callback is defined) should call that function at some point, using the implementation provided by the specific module (the one that implements the behavior).

Reply:
Did the explanation adjustment, check-n-tell!